src/pages/experiments/[id].tsx

```diff
@@ -118,6 +45,10 @@ export default function Experiment() {
     }
   }, [updateMutation, experiment.data?.id, experiment.data?.label, label]);
 
+  useEffect(() => {
```

Comment:
Was this moved just because you found the same bug before I merged https://github.com/OpenPipe/OpenPipe/pull/88?

Reply:
Yes, I'll move it back where you put it to make the diff cleaner